docs/CHANGELOG.asciidoc

```diff
@@ -82,6 +83,167 @@ Review important information about the {kib} 8.x releases.
 include::upgrade-notes.asciidoc[]
 
 
+[[release-notes-8.17.0]]
+== {kib} 8.17.0
+
+coming::[8.17.0]
+
+For information about the {kib} 8.17.0 release, review the following information.
+
+The 8.17.0 release includes the following known issues.
+
+[float]
+[[known-issues-8.17.0]]
+=== Known issues
+
+[discrete]
+[[known-203553]]
```

Comment:
We no longer need this known issue - it was fixed in the last build candidate. Thanks!

Reply:
@ElenaStoeva Okay, I've removed it. Can you please approve the changes?